ArchUnit.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26430.16
+# Visual Studio Version 16
```

Comment:
I'm Sorry, it looks like VS updated the Solution File on it's own.
It was not shown in the commits.

Reply:
I'm sorry, but it didn't passed the automatic tests.
It might be due the changes in the solution file VS made on it's own (which was not shown in the commits)
and I had problems with the global.json file, which I had to clear so that the tests run.

I try my best to fix these problems.

Kind Regards:
 - Joneleit, Stephan